### PR TITLE
Names and icons clickable

### PR DESCRIPTION
**Pull request by victor-dantas**
bot's name and avatar are clickable

### CODE REVIEW

**Comment by victor-dantas** [comment]
fix #248

File: src/components/repository/RepositoryInfo.vue [hunk 1]
```diff
@@ -2,20 +2,26 @@
...
-          <bh-icon
+          <bh-icon-button
             value="botinho"
-            class="repository-info__title__bagde__icon" />
+            class="repository-info__title__bagde__icon"
+            @click="redirect()" />
         </span>
-        <span>{{ repository.name }}</span>
+        <div
+          class="repository-info__cursor-pointer"
```
Comment (douglasppaz) [inline src/components/repository/RepositoryInfo.vue]: BH has a class called clickable.. Remove this class definition and set .clickable here.

File: src/components/repository/RepositoryInfo.vue [hunk 2]
```diff
@@ -2,20 +2,26 @@
   <div class="bh-grid">
     <div class="bh-grid__item bh-grid__item--grow-0 repository-info__big-badge-wrapper">
       <div class="repository-info__big-badge">
-        <bh-icon
+        <bh-icon-button
           size="medium"
           value="botinho"
-          class="repository-info__big-badge__icon" />
+          class="repository-info__big-badge__icon"
+          @click="redirect()" />
```
Comment (douglasppaz) [inline src/components/repository/RepositoryInfo.vue]: Use route link here.
Like here: https://github.com/Ilhasoft/bothub-webapp/blob/1caa1559aed8dfbc30f9fa69b0ecc9448fe23e36/src/components/repository/RepositoryInfo.vue#L27

File: src/components/repository/RepositoryInfo.vue [hunk 3]
```diff
@@ -2,20 +2,26 @@
...
     </div>
     <div class="bh-grid__item">
       <div class="repository-info__title">
         <span class="repository-info__title__bagde">
-          <bh-icon
+          <bh-icon-button
             value="botinho"
-            class="repository-info__title__bagde__icon" />
+            class="repository-info__title__bagde__icon"
+            @click="redirect()" />
```
Comment (douglasppaz) [inline src/components/repository/RepositoryInfo.vue]: rout link here too

File: src/components/repository/RepositoryInfo.vue [hunk 4]
```diff
@@ -74,6 +80,17 @@ export default {
       'getProfile',
     ]),
   },
+  methods: {
+    redirect() {
```
Comment (douglasppaz) [inline src/components/repository/RepositoryInfo.vue]: Using route link you don't need use this class

File: src/components/repository/RepositoryInfo.vue [hunk 5]
```diff
@@ -116,6 +137,10 @@ export default {
     font-weight: $font-weight-bolder;
     margin-bottom: .5rem;
 
+    &__name {
+      color: $color-bolder-grey
```
Comment (douglasppaz) [inline src/components/repository/RepositoryInfo.vue]: ;

File: src/components/repository/RepositoryInfo.vue [hunk 6]
```diff
@@ -116,6 +137,10 @@ export default {
     font-weight: $font-weight-bolder;
     margin-bottom: .5rem;
 
+    &__name {
```
Comment (douglasppaz) [inline src/components/repository/RepositoryInfo.vue]: Remove this class definition, use BH .text-color-bolder-grey

File: src/components/repository/RepositoryInfo.vue [hunk 7]
```diff
@@ -2,20 +2,41 @@
...
         </span>
-        <span>{{ repository.name }}</span>
+        <router-link
+          :to="{
+            name: 'repository-home',
+            params: {
+              ownerNickname: repository.owner__nickname,
+              slug: repository.slug,
+            },
+        }"><span class="repository-info__title__name">{{ repository.name }}</span></router-link>
```
Comment (douglasppaz) [inline src/components/repository/RepositoryInfo.vue]: Use bh class .text-color-bolder-grey

File: src/components/repository/RepositoryInfo.vue [hunk 8]
```diff
@@ -73,6 +69,16 @@ export default {
...
     ]),
+    routerParameters() {
+      const to = {
+        name: 'repository-home',
+        params: {
+          ownerNickname: this.repository.owner__nickname,
+          slug: this.repository.slug,
+        },
+      };
+      return to;
```
Comment (douglasppaz) [inline src/components/repository/RepositoryInfo.vue]: You don't need create variable, return directly return {};

**Pull request merged**